frontend/components/DuccStatsList.tsx

```diff
@@ -35,6 +42,6 @@ interface IProps {
 }
 
 export enum ScoreType {
-  FRIENDS = 'duccFriends',
-  KILLERS = 'duccKillers'
-}
\ No newline at end of file
+  FRIENDS = "duccFriends",
```

Comment:
So are we using single quotes in the backend and double quotes in the frontend now? lol

Reply:
blame prettier